netlify.sh

```diff
@@ -21,29 +22,39 @@ PID_MONGO=$!
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "${DIR}"
 
-. "$(curl -fsSL --retry 5 --retry-max-time 120 --connect-timeout 5 --speed-time 30 --speed-limit 1024 https://dlang.org/install.sh | bash -s install "dmd-${DMD_VERSION}" --activate)"
+. "$(curl "${CURL_FLAGS[@]}" https://dlang.org/install.sh | bash -s install "dmd-${DMD_VERSION}" --activate)"
+
+curl "${CURL_FLAGS[@]}" https://code.dlang.org/api/packages/dump | gunzip > mirror.json
```

Comment:
We could also upload a JSON dump somewhere else if this becomes a problem, but for now there aren't too many PRs at dub-registry so having a separate file for this will probably just be more work.